mail_activity_team/tests/test_mail_activity_team.py

```diff
@@ -116,3 +116,11 @@ def test_team_onchanges(self):
             self.team2.member_ids = [(3, self.employee.id)]
             self.team2._onchange_member_ids()
             self.assertFalse(self.team2.user_id)
+
+    def test_activity_count(self):
+        res = self.env['res.users'].with_context(
+            {'team_activities': True}
+        ).activity_user_count(
```

Comment:
And here use the new function name to get travis green `systray_get_activities`.
LGTM the new name.